rpc/internal/test_results_service.go

```diff
@@ -83,16 +84,63 @@ func (s *testResultsService) AddTestResults(ctx context.Context, results *TestRe
 	}
 
 	if !record.Info.HistoricalDataDisabled {
-		for _, res := range exportedResults {
-			// TODO: Remove this job with EVG-14440.
-			j := units.NewHistoricalTestDataJob(s.env, record.Info, res)
-			j.Run(ctx)
-			grip.Error(message.WrapError(j.Error(), message.Fields{
-				"message":     "failed to run historical test data job",
-				"info":        record.Info,
-				"test_result": res,
-			}))
-		}
+		go func() {
+			defer func() {
+				if err := recovery.HandlePanicWithError(recover(), nil, "historical test data update"); err != nil {
+					grip.Error(message.WrapError(err, message.Fields{
+						"message":           "failed to update historical test data",
+						"test_results_info": record.Info,
+					}))
+				}
+			}()
+
+			htdCtx, cancel := context.WithTimeout(context.Background(), time.Minute)
+			defer cancel()
+
+			conf := model.NewCedarConfig(s.env)
+			if err := conf.Find(); err != nil {
+				grip.Error(message.WrapError(errors.Wrap(err, "finding cedar configuration"), message.Fields{
+					"message":           "failed to update historical test data",
+					"test_results_info": record.Info,
+				}))
+				return
+			}
+			if conf.Flags.DisableHistoricalTestData {
```

Comment:
This is effectively dropping data points in the historical test stats. Can this log when stats are disabled and what data point it's dropping?

Reply:
That may be excessive logging since this is an application wide flag that would result in nearly every call to `AddTestResults` logging this same thing.